src/Utilities.js

```diff
@@ -16,3 +21,18 @@ export const resourceToName = (uri) => {
     return undefined
   }
 }
+
+export const generateTitle = ( property ) => {
```

Comment:
This function is throwing me a bit. (Sorry, late on a Friday...) This function can return a component, a string, or an array? I am wondering if that suggests these should be different functions. That way, calling code can act on the return value with more confidence in what it's getting back.

Reply:
Thanks, this comment made me look at how things were being rendered and I did more refactoring. The `<PropertyRemark>` is going to become a component named `<PropertyTitle>` that will render the right kind of title and the Utilities will only perhaps host a `isRequiredSuperscript()` function. Stay tuned...

Reply:
see comment in tests re name of this method.  Title, to me, is too vague, and what it returns is confusing as well.  "propertyComponentHeading" ?  I remain a bit confused, sorry.  Maybe I need to look at more than the diffs here?  Or maybe the name of this function could match how it's used a bit better -- seems to be a prop-remark prop in `<PropertyRemark>`

Reply:
This function is removed. Now the `PropertyTitle` component is responsible for rendering the right kind of title.